WooCommerce/Classes/ViewRelated/Orders/Order Creation/CustomAmounts/OrderCustomAmountsSection.swift

```diff
@@ -86,55 +102,122 @@ struct OrderCustomAmountsSection: View {
-                .presentationDetents([.height(218)])
-                .presentationDragIndicator(.visible)
-        } else {
-            optionsBottomSheetContent
+        optionsBottomSheetContent
+            .presentationDetents(detentsForOptionsBottomSheet)
+            .presentationDragIndicator(.visible)
+    }
+
+    private var detentsForOptionsBottomSheet: Set<PresentationDetent> {
+        switch dynamicTypeSize {
+        case .xSmall, .small, .medium, .large:
+            return [.height(175), .medium]
+        case .xLarge, .xxLarge:
+            return [.height(200), .medium]
+        case .xxxLarge:
+            return [.height(230), .medium]
+        case .accessibility1, .accessibility2, .accessibility3, .accessibility4, .accessibility5:
+            return [.medium, .large]
+        @unknown default:
+            return [.large]
         }
     }
 
     @ViewBuilder private var optionsBottomSheetContent: some View {
         VStack (alignment: .leading, spacing: Layout.optionsBottomSheetContentVerticalSpacing) {
             Text(Localization.optionsDialogAddCustomAmountTitle)
                 .subheadlineStyle()
-                .padding(.top, Layout.optionsBottomSheetContentTitleTopPadding)
-                .padding(.bottom, Layout.optionsBottomSheetContentTitleBottomPadding)
-
-            HStack {
-                Text("$")
-                    .frame(minWidth: Layout.optionsBottomSheetButtonSymbolWidth)
-                    .padding(.trailing, Layout.optionsBottomSheetButtonSymbolTrailing)
+                .fixedSize(horizontal: false, vertical: true)
+                .padding(.top, Layout.optionsBottomSheetContentVerticalSpacing)
+                .padding([.leading, .trailing], Layout.optionsBottomSheetPadding)
 
-                Button(Localization.optionsDialogFixedAmountButtonTitle) {
+            List {
+                Button {
                     addCustomAmountOption = .fixedAmount
                     showAddCustomAmountsAfterOptionsDialog()
-
+                } label: {
+                    optionLabel(symbol: sectionViewModel.currencySymbol,
+                                title: Localization.optionsDialogFixedAmountButtonTitle)
                 }
-                .bodyStyle()
+                .listRowSeparator(.hidden)
                 .accessibilityIdentifier(Accessibility.fixedAmountIdentifier)
-            }
-            .padding(.bottom, Layout.optionsBottomSheetContentVerticalSpacing)
-
-            HStack {
-                Text("%")
-                    .frame(minWidth: Layout.optionsBottomSheetButtonSymbolWidth)
-                    .padding(.trailing, Layout.optionsBottomSheetButtonSymbolTrailing)
 
-                Button(Localization.optionsDialogPercentageButtonTitle) {
+                Button {
                     addCustomAmountOption = .orderTotalPercentage
                     showAddCustomAmountsAfterOptionsDialog()
+                } label: {
+                    optionLabel(symbol: "%",
```

Comment:
Question: Should we switch this string for SFSymbol's `.percent` ?

Reply:
I considered this, but since it's displayed under the currency symbol as text, it seems like it'll look more consistent this way.

Changing the currency symbols to SF symbols has various issues: not all currencies are represented, and just having the parsing code is annoying.